lib/src/tar_encoder.dart

```diff
@@ -16,7 +16,7 @@ class TarEncoder {
 
     finish();
 
-    return output_stream.getBytes();
+    return Uint8List.fromList(output_stream.getBytes());
```

Comment:
output_stream.getBytes() is a Uint8ListView, it can be typecast as Uint8List, without making a full copy of the bytes.

Reply:
Note that List<int> is an abstract interface, which Uint8List is an implementation of. Everything in the library that refers to LIst<int> is already Uint8List, it just used the base interface name of List<int> because I thought at the time it would be better to not expose the typed_data names outside the library. But that was probably a mistake, it causes some confusion about what the data actually is.